### DISCUSSION TITLE
Color change "my routes" not effective

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#369] [Q&A] [answered]
If I change the color of the route under: “My Places/Routes/…”, nothing changes. The old color remains.
The color change will only take effect after you restart Cruiser.

**Comment by DolomitesTom** [comment]
Which version? Last version 3.0.33 b17 it works fine. You have to push OK in order to save the change.

**Reply by Leeb64** [reply]
Version 3.0.33b17
Doesn't work (for me)!
The color of the small symbol changes, but not the color of the route.
The change is only visible after Cruiser is closed and restarted.
It's the same with my other smartphone!

**Comment by devemux86** [comment]
@Leeb64 thanks for the report.

I can reproduce it and will fix it. :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
